corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -602,6 +621,9 @@ cloudCare.AppView = Backbone.View.extend({
 
         return url;
     },
+    getSyncUrl: function () {
+        return "/webforms/player_proxy";
```

Comment:
normally there's an abstraction boundary between the player_proxy and this code. only the adapter `formnav-all.js` posts to the `player_proxy`, and this code in general does not know about it. i think for this case it's OK to break it since syncing should be outside of the form session